pkg/api/persistentvolume/util.go

```diff
@@ -39,7 +39,7 @@ func VisitPVSecretNames(pv *api.PersistentVolume, visitor func(string, string) b
 		}
 		return true
 	case source.CephFS != nil:
-		if source.CephFS.SecretRef != nil && !visitor(getClaimRefNamespace(pv), source.CephFS.SecretRef.Name) {
+		if source.CephFS.SecretRef != nil && !visitor(source.CephFS.SecretRef.Namespace, source.CephFS.SecretRef.Name) {
```

Comment:
this isn't accurate... if `source.CephFS.SecretRef.Namespace` is empty (which it will be for previously persisted PV objects), and `pv.Spec.ClaimRef` is non-nil, this should fall back to `pv.Spec.ClaimRef.Namespace`